lib/utils/pathutils.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var join = require('join-path');
+var _ = require('lodash');
+
+var INDEX_FILE = 'index.html';
+
+var pathutils = {
+  asDirectoryIndex: function(p) {
```

Comment:
nit: not generally a fan of single-char variables. `path` isn't too overloaded here